runtime/vimrc_example.vim

```diff
@@ -1,8 +1,8 @@
 " An example for a vimrc file.
 "
 " To use it, copy it to
-"     for Unix:  ~/.vimrc
-"     for Windows:  $VIM\_vimrc
+"     for Unix:  ${XDG_CONFIG_HOME:=$HOME/.config}/nvim/init.vim
```

Comment:
For most users, `${XDG_CONFIG_HOME:=$HOME/.config}` is just confusing and pedantic. Users who are just getting started are almost certainly not going to care about XDG, so `~/.config` is sufficient as a general assumption.